### PR TITLE
[ADR-0005] support component styling with tailwind css in design system

### PR DESCRIPTION
**Pull request by KhudaDad414**
I purpose we support styling components in design system with tailwind css.

### CODE REVIEW

**Review by Amzani — CHANGES_REQUESTED** [review]

**Review by Amzani — APPROVED** [review]
LGTM

**Comment by KhudaDad414** [comment]
@Amzani can't merge without the code-owner's approval. 😄

**Pull request merged**
